akka/src/main/scala/com/qvantel/jsonapi/akka/JsonApiSupport.scala

```diff
@@ -62,6 +63,10 @@ trait JsonApiSupport extends JsonApiSupport0 {
         value.entity match {
           case HttpEntity.Strict(_, data)     => extractEntities(data, include)
           case HttpEntity.Default(_, _, data) => extractEntities(data, include)
+          case entity @ HttpEntity.Chunked(_, _) =>
```

Comment:
Doesn’t seem like you need to be switching on the entity type at all since you need to read the entire thing in any case. Perhaps just do `.toStrict` or do something like what [`SprayJsonSupport`](https://github.com/akka/akka-http/blob/master/akka-http-marshallers-scala/akka-http-spray-json/src/main/scala/akka/http/scaladsl/marshallers/sprayjson/SprayJsonSupport.scala#L25) does?